packages/prettier-plugin-java/test/unit-test/bug-fixes/_input.java

```diff
@@ -7,6 +7,10 @@ void process(
         Map.@NonNull Entry<String, SkeletonConfiguration> entry,
         @NonNull Map<String, Object> context
     ) {}
+
+    void process() {
+        coco.tata[]()();
```

Comment:
This isn't valid Java syntax, right? Does something like this still test the edge case you were hoping to account for?
```suggestion
        coco.tata[0]();
```

Reply:
You're right, I was too much on what was allowed by java-parser ! I removed this test case as it won't happen in real life.